src/cascadia/TerminalSettingsModel/ActionMap.idl

```diff
@@ -14,10 +14,13 @@ namespace Microsoft.Terminal.Settings.Model
         [method_name("GetKeyBindingForActionWithArgs")] Microsoft.Terminal.Control.KeyChord GetKeyBindingForAction(ShortcutAction action, IActionArgs actionArgs);
 
         Windows.Foundation.Collections.IMapView<String, Command> NameMap { get; };
-        Windows.Foundation.Collections.IMapView<Microsoft.Terminal.Control.KeyChord, Command> GlobalHotkeys();
+        Windows.Foundation.Collections.IMapView<Microsoft.Terminal.Control.KeyChord, Command> KeyBindings { get; };
+        Windows.Foundation.Collections.IMapView<Microsoft.Terminal.Control.KeyChord, Command> GlobalHotkeys { get; };
     };
 
     [default_interface] runtimeclass ActionMap : IActionMapView
     {
+        Boolean RebindKeys(Microsoft.Terminal.Control.KeyChord oldKeys, Microsoft.Terminal.Control.KeyChord newKeys);

```

Comment:
i'm surprised that this takes the oldkeys and not the action -- shouldn't we enforce that the caller is providing an action?

Reply:
So, `oldKeys` can be used as a primary key the way it's written now.

If we wanted to switch over to using `ActionAndArgs`, we could make the API look more like this:
```cpp
void RegisterKeyBinding(ActionAndArgs, KeyChord);
void EraseKeyBinding(ActionAndArgs, KeyChord);

// For later...
void SetActionForKeyBinding(KeyChord, ActionAndArgs);
void SetName(ActionAndArgs, String);
```

I'm intentionally trying to make it so that we don't work with `Command` outside of `ActionMap`.